the-list-be/server.js

```diff
@@ -12,7 +12,11 @@ const db = client.db('database');
 const students = db.collection('students');
 
 router.post('/students', async ctx => {
-  const insertResult = await students.insertOne(ctx.request.body);
+  const insertBody = ctx.request.body;
+  const name = insertBody.name;
```

Comment:
Иначе упадёт тут, если `insertBody` undefined.

```suggestion
  const name = insertBody?.name;
```